terraform/aws/versions.tf

```diff
@@ -18,3 +18,7 @@ terraform {
   }
   required_version = ">= 0.13"
 }
+
+variable "eks_kubernetes_version" {
+  default = "1.20"
```

Comment:
Most recent version supported by AWS:

https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html

We should be good until June 2022. Not that long though 🤔